.gitignore

```diff
@@ -15,4 +15,12 @@ $RECYCLE.BIN/
 *.msp
 
 # Local config AChecker
-include/config.inc.php
\ No newline at end of file
+include/config.inc.php
+
+<<<<<<< HEAD
```

Comment:
Line 20-26 is only relevant to you local development environment and should be removed.